x-pack/plugins/canvas/canvas_plugin_src/functions/common/containerStyle.ts

```diff
@@ -58,13 +58,14 @@ export function containerStyle(): NullContextFunction<'containerStyle', Containe
         options: ['repeat-x', 'repeat', 'space', 'round', 'no-repeat', 'space'],
       },
       opacity: {
-        types: ['number', 'null'],
+        types: ['number'],
         help: argHelp.opacity,
       },
       overflow: {
         types: ['string'],
         help: argHelp.overflow,
         options: ['visible', 'hidden', 'scroll', 'auto'],
+        default: 'hidden',
```

Comment:
q: Seems like a reasonable default, but I'm curious if you added to fix something or just because you thought it was a good value?

Reply:
The CSS for `canvasElement` sets `overflow: hidden`, and this is the default style applied to all Canvas elements if it's not set to something else in `containerStyle`, so I thought it'd be good to set it as the default here just to be explicit.